prov/tcp/src/tcpx.h

```diff
@@ -271,7 +271,7 @@ struct tcpx_fabric {
 	struct util_fabric	util_fabric;
 };
 
-#define TCPX_INTERNAL_MASK	GENMASK_ULL(63, 58)
+#define TCPX_INTERNAL_MASK	GENMASK_ULL(63, 58) | FI_COMPLETION
```

Comment:
Add the parenthesis here, so that callers don't need to add it (which isn't an obvious thing to do), and remove it from the change below.